tests/e2e/src/pages/main.page.ts

```diff
@@ -77,6 +81,10 @@ export class MainPage extends BasePage {
     return `${this.byTestId}network-switcher`;
   }
 
+  get proceedButton() {
+    return "//*[contains(., 'Proceed')]";
```

Comment:
I would recommend to specify any button selector if this is only about a button: //button[text()[contains(string(), '${name}')]].  Perhaps it make sense to call it just aButton()
` async aButton(name: string) {
    return "//button[text()[contains(string(), '${name}')]]";
    }
    `